int32 robot_ID
float64 x
float64 y
